<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Harshal" ignored="0" name="ProductPage Validation" tests="40" failures="15" timestamp="2021-10-17T17:07:48 IST" time="182.222" errors="0">
  <testcase name="CartAddDate" time="6.608" classname="com.plp.tests.ProductPageValidation">
    <failure type="java.lang.AssertionError" message="Website is accepting previous delivery date">
      <![CDATA[java.lang.AssertionError: Website is accepting previous delivery date
at com.plp.tests.ProductPageValidation.CartAddDate(ProductPageValidation.java:160)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- CartAddDate -->
  <testcase name="ProductNavigation" time="4.638" classname="com.plp.tests.ProductPageValidation"/>
  <testcase name="ProductNavigation" time="4.091" classname="com.plp.tests.ProductPageValidation"/>
  <testcase name="ProductNavigation" time="3.32" classname="com.plp.tests.ProductPageValidation"/>
  <testcase name="ProductNavigation" time="3.385" classname="com.plp.tests.ProductPageValidation"/>
  <testcase name="ProductNavigation" time="3.437" classname="com.plp.tests.ProductPageValidation">
    <failure type="java.lang.AssertionError" message="expected [sdf] but found [Canon EOS 5D]">
      <![CDATA[java.lang.AssertionError: expected [sdf] but found [Canon EOS 5D]
at com.plp.tests.ProductPageValidation.ProductNavigation(ProductPageValidation.java:38)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- ProductNavigation -->
  <testcase name="ProductNavigation" time="3.498" classname="com.plp.tests.ProductPageValidation"/>
  <testcase name="ReviewFunc" time="5.424" classname="com.plp.tests.ProductPageValidation"/>
  <testcase name="ReviewFunc" time="5.48" classname="com.plp.tests.ProductPageValidation"/>
  <testcase name="ReviewFunc" time="5.108" classname="com.plp.tests.ProductPageValidation"/>
  <testcase name="ReviewFunc" time="5.072" classname="com.plp.tests.ProductPageValidation"/>
  <testcase name="ReviewFunc" time="5.191" classname="com.plp.tests.ProductPageValidation"/>
  <testcase name="ReviewFunc" time="5.077" classname="com.plp.tests.ProductPageValidation"/>
  <testcase name="VerifyAddToCart" time="5.344" classname="com.plp.tests.ProductPageValidation"/>
  <testcase name="VerifyAddToCart" time="5.027" classname="com.plp.tests.ProductPageValidation"/>
  <testcase name="VerifyAddToCart" time="4.577" classname="com.plp.tests.ProductPageValidation"/>
  <testcase name="VerifyAddToCart" time="4.439" classname="com.plp.tests.ProductPageValidation"/>
  <testcase name="VerifyAddToCart" time="6.667" classname="com.plp.tests.ProductPageValidation">
    <failure type="java.lang.AssertionError" message="Product is not added to the cart">
      <![CDATA[java.lang.AssertionError: Product is not added to the cart
at com.plp.tests.ProductPageValidation.VerifyAddToCart(ProductPageValidation.java:129)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- VerifyAddToCart -->
  <testcase name="VerifyAddToCart" time="5.194" classname="com.plp.tests.ProductPageValidation"/>
  <testcase name="VerifyFields" time="5.145" classname="com.plp.tests.ProductPageValidation">
    <failure type="java.lang.AssertionError" message="First Name Field is Accepting Numeric Data">
      <![CDATA[java.lang.AssertionError: First Name Field is Accepting Numeric Data
at com.plp.tests.ProductPageValidation.VerifyFields(ProductPageValidation.java:115)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- VerifyFields -->
  <testcase name="VerifyReviews" time="4.157" classname="com.plp.tests.ProductPageValidation">
    <failure type="java.lang.AssertionError" message="Reviews are not available">
      <![CDATA[java.lang.AssertionError: Reviews are not available
at com.plp.tests.ProductPageValidation.VerifyReviews(ProductPageValidation.java:80)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- VerifyReviews -->
  <testcase name="VerifyReviews" time="4.638" classname="com.plp.tests.ProductPageValidation">
    <failure type="java.lang.AssertionError" message="Reviews are not available">
      <![CDATA[java.lang.AssertionError: Reviews are not available
at com.plp.tests.ProductPageValidation.VerifyReviews(ProductPageValidation.java:80)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- VerifyReviews -->
  <testcase name="VerifyReviews" time="4.021" classname="com.plp.tests.ProductPageValidation">
    <failure type="java.lang.AssertionError" message="Reviews are not available">
      <![CDATA[java.lang.AssertionError: Reviews are not available
at com.plp.tests.ProductPageValidation.VerifyReviews(ProductPageValidation.java:80)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- VerifyReviews -->
  <testcase name="VerifyReviews" time="3.891" classname="com.plp.tests.ProductPageValidation">
    <failure type="java.lang.AssertionError" message="Reviews are not available">
      <![CDATA[java.lang.AssertionError: Reviews are not available
at com.plp.tests.ProductPageValidation.VerifyReviews(ProductPageValidation.java:80)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- VerifyReviews -->
  <testcase name="VerifyReviews" time="4.062" classname="com.plp.tests.ProductPageValidation">
    <failure type="java.lang.AssertionError" message="Reviews are not available">
      <![CDATA[java.lang.AssertionError: Reviews are not available
at com.plp.tests.ProductPageValidation.VerifyReviews(ProductPageValidation.java:80)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- VerifyReviews -->
  <testcase name="VerifyReviews" time="4.414" classname="com.plp.tests.ProductPageValidation">
    <failure type="java.lang.AssertionError" message="Reviews are not available">
      <![CDATA[java.lang.AssertionError: Reviews are not available
at com.plp.tests.ProductPageValidation.VerifyReviews(ProductPageValidation.java:80)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- VerifyReviews -->
  <testcase name="verifyDescription" time="3.321" classname="com.plp.tests.ProductPageValidation"/>
  <testcase name="verifyDescription" time="4.079" classname="com.plp.tests.ProductPageValidation"/>
  <testcase name="verifyDescription" time="3.252" classname="com.plp.tests.ProductPageValidation"/>
  <testcase name="verifyDescription" time="3.432" classname="com.plp.tests.ProductPageValidation"/>
  <testcase name="verifyDescription" time="3.424" classname="com.plp.tests.ProductPageValidation"/>
  <testcase name="verifyDescription" time="3.611" classname="com.plp.tests.ProductPageValidation"/>
  <testcase name="verifyMandatoryFields" time="3.83" classname="com.plp.tests.ProductPageValidation"/>
  <testcase name="verifyQuantity" time="5.351" classname="com.plp.tests.ProductPageValidation"/>
  <testcase name="verifySpecification" time="0.016" classname="com.plp.tests.ProductPageValidation">
    <failure type="org.openqa.selenium.UnhandledAlertException" message="unexpected alert open: {Alert text : SyntaxError: Unexpected token &amp;lt; in JSON at position 0
parsererror
&amp;lt;b&amp;gt;Warning&amp;lt;/b&amp;gt;: unlink(/home/ocdemo/storage/cache/cache.catalog.language.1634473989): No such file or directory in &amp;lt;b&amp;gt;/home/ocdemo/public_html/system/library/cache/file.php&amp;lt;/b&amp;gt; on line &amp;lt;b&amp;gt;68&amp;lt;/b&amp;gt;{&amp;quot;success&amp;quot;:&amp;quot;Success: You have added &amp;lt;a href=\&amp;quot;https:\/\/demo.opencart.com\/index.php?route=product\/product&amp;amp;amp;product_id=47\&amp;quot;&amp;gt;HP LP3065&amp;lt;\/a&amp;gt; to your &amp;lt;a href=\&amp;quot;https:\/\/demo.opencart.com\/index.php?route=checkout\/cart\&amp;quot;&amp;gt;shopping cart&amp;lt;\/a&amp;gt;!&amp;quot;,&amp;quot;total&amp;quot;:&amp;quot;6 item(s) - $2,411.99&amp;quot;}}
  (Session info: chrome=94.0.4606.81): SyntaxError: Unexpected token &amp;lt; in JSON at position 0
parsererror
&amp;lt;b&amp;gt;Warning&amp;lt;/b&amp;gt;: unlink(/home/ocdemo/storage/cache/cache.catalog.language.1634473989): No such file or directory in &amp;lt;b&amp;gt;/home/ocdemo/public_html/system/library/cache/file.php&amp;lt;/b&amp;gt; on line &amp;lt;b&amp;gt;68&amp;lt;/b&amp;gt;{&amp;quot;success&amp;quot;:&amp;quot;Success: You have added &amp;lt;a href=\&amp;quot;https:\/\/demo.opencart.com\/index.php?route=product\/product&amp;amp;amp;product_id=47\&amp;quot;&amp;gt;HP LP3065&amp;lt;\/a&amp;gt; to your &amp;lt;a href=\&amp;quot;https:\/\/demo.opencart.com\/index.php?route=checkout\/cart\&amp;quot;&amp;gt;shopping cart&amp;lt;\/a&amp;gt;!&amp;quot;,&amp;quot;total&amp;quot;:&amp;quot;6 item(s) - $2,411.99&amp;quot;}
Build info: version: &amp;apos;3.141.59&amp;apos;, revision: &amp;apos;e82be7d358&amp;apos;, time: &amp;apos;2018-11-14T08:17:03&amp;apos;
System info: host: &amp;apos;HARSHAL&amp;apos;, ip: &amp;apos;192.168.56.1&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;1.8.0_251&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 94.0.4606.81, chrome: {chromedriverVersion: 94.0.4606.61 (418b78f5838ed..., userDataDir: C:\Users\Harshal\AppData\Lo...}, goog:chromeOptions: {debuggerAddress: localhost:6553}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 6ebcfb467ecbe960e116ac89a55747e3
*** Element info: {Using=name, value=search}">
      <![CDATA[org.openqa.selenium.UnhandledAlertException: unexpected alert open: {Alert text : SyntaxError: Unexpected token < in JSON at position 0
parsererror
<b>Warning</b>: unlink(/home/ocdemo/storage/cache/cache.catalog.language.1634473989): No such file or directory in <b>/home/ocdemo/public_html/system/library/cache/file.php</b> on line <b>68</b>{"success":"Success: You have added <a href=\"https:\/\/demo.opencart.com\/index.php?route=product\/product&amp;product_id=47\">HP LP3065<\/a> to your <a href=\"https:\/\/demo.opencart.com\/index.php?route=checkout\/cart\">shopping cart<\/a>!","total":"6 item(s) - $2,411.99"}}
  (Session info: chrome=94.0.4606.81): SyntaxError: Unexpected token < in JSON at position 0
parsererror
<b>Warning</b>: unlink(/home/ocdemo/storage/cache/cache.catalog.language.1634473989): No such file or directory in <b>/home/ocdemo/public_html/system/library/cache/file.php</b> on line <b>68</b>{"success":"Success: You have added <a href=\"https:\/\/demo.opencart.com\/index.php?route=product\/product&amp;product_id=47\">HP LP3065<\/a> to your <a href=\"https:\/\/demo.opencart.com\/index.php?route=checkout\/cart\">shopping cart<\/a>!","total":"6 item(s) - $2,411.99"}
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'HARSHAL', ip: '192.168.56.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_251'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 94.0.4606.81, chrome: {chromedriverVersion: 94.0.4606.61 (418b78f5838ed..., userDataDir: C:\Users\Harshal\AppData\Lo...}, goog:chromeOptions: {debuggerAddress: localhost:6553}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 6ebcfb467ecbe960e116ac89a55747e3
*** Element info: {Using=name, value=search}
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:120)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:323)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByName(RemoteWebDriver.java:404)
at org.openqa.selenium.By$ByName.findElement(By.java:284)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at com.sun.proxy.$Proxy10.clear(Unknown Source)
at com.automation.utils.SearchProduct.SearchProductFunc(SearchProduct.java:159)
at com.plp.tests.ProductPageValidation.verifySpecification(ProductPageValidation.java:60)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- verifySpecification -->
  <testcase name="verifySpecification" time="4.364" classname="com.plp.tests.ProductPageValidation">
    <failure type="java.lang.AssertionError" message="Specification field Not Active">
      <![CDATA[java.lang.AssertionError: Specification field Not Active
at com.plp.tests.ProductPageValidation.verifySpecification(ProductPageValidation.java:66)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- verifySpecification -->
  <testcase name="verifySpecification" time="3.678" classname="com.plp.tests.ProductPageValidation"/>
  <testcase name="verifySpecification" time="3.522" classname="com.plp.tests.ProductPageValidation">
    <failure type="java.lang.AssertionError" message="Specification field Not Active">
      <![CDATA[java.lang.AssertionError: Specification field Not Active
at com.plp.tests.ProductPageValidation.verifySpecification(ProductPageValidation.java:66)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- verifySpecification -->
  <testcase name="verifySpecification" time="3.398" classname="com.plp.tests.ProductPageValidation">
    <failure type="java.lang.AssertionError" message="Specification field Not Active">
      <![CDATA[java.lang.AssertionError: Specification field Not Active
at com.plp.tests.ProductPageValidation.verifySpecification(ProductPageValidation.java:66)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- verifySpecification -->
  <testcase name="verifySpecification" time="3.913" classname="com.plp.tests.ProductPageValidation">
    <failure type="java.lang.AssertionError" message="Specification field Not Active">
      <![CDATA[java.lang.AssertionError: Specification field Not Active
at com.plp.tests.ProductPageValidation.verifySpecification(ProductPageValidation.java:66)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- verifySpecification -->
</testsuite> <!-- ProductPage Validation -->
